addons/vehicle_damage/functions/fnc_medicalDamage.sqf

```diff
@@ -0,0 +1,44 @@
+#include "..\script_component.hpp"
+/*
+ * Author: johnb43
+ * Applies medical damage to a unit.
+ *
+ * Arguments:
+ * 0: Target <OBJECT>
+ * 1: Source <OBJECT>
+ * 2: Instigator <OBJECT>
+ * 3: Guarantee death? <BOOL> (default: false)
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [cursorObject, player, player] call grenades_hatches_main_fnc_medicalDamage;
```

Comment:
```suggestion
 * [cursorObject, player, player] call ace_vehicle_damage_main_fnc_medicalDamage;
```

Reply:
Oops, forgot to remove the header from my mod...